src/main/webapp/app/guided-tour/guided-tour.component.html

```diff
@@ -0,0 +1,69 @@
+<div
```

Comment:
I think it would be good to extract 2-3 components from this component.

Reply:
@thilo-behnke I removed all orb markup (that we don't use). The component should be oversee-able now, isn't it?